components/styled/table/src/atoms/table.tsx

```diff
@@ -8,6 +8,18 @@ const Container = styled.div`
   background-color: white;
   border-radius: 6px;
   display: inline-block;
+  max-width: calc(100%);
+  max-height: calc(100%);
+  overflow: scroll;
+  &::-webkit-scrollbar {
+    width: 7px;
+    height: 7px;
+    border: 0;
+  }
+  &::-webkit-scrollbar-thumb {
+    background: rgb(190, 190, 190);
```

Comment:
Optional - generally we _should_ use values from @ltht-react/styles (`packages/styles/src/atoms/colours.ts`) for color wherever possible, there's quite a few grey values. Alternatively, define your own in there so we can standardise